### ISSUE TITLE
Multi-label classification

### ISSUE THREAD

**Issue by andreassot10** [CDU-data-science-team/pxtextmining#25]
https://towardsdatascience.com/multi-label-text-classification-with-scikit-learn-30714b7819c5?gi=f2f82816a91e

Make dedicated branch for this. It may make sense to add choice between multi-class and multi-label as a pipeline function argument.

**Comment by ChrisBeeley** [comment]
I'd like to give this a try some time if possible @asegun-cod - we'll discuss in the week

**Comment by yiwen-h** [comment]
Think the new multilabel dataset will need a completely separate pipeline. 

Will need to think about ways of dealing with imbalanced dataset. Old method was to use random oversampling from imblearn (will this work with multilabel? perhaps it will create false relationships between categories if not careful), another option might be https://github.com/makcedward/nlpaug ? 

Also need to research performance metrics for multilabel. write everything up in project documentation to make decision making transparent.

models to try:
- sklearn options: https://scikit-learn.org/stable/modules/multiclass.html
- huggingface transformers? (bear in mind long training times)

**Comment by yiwen-h** [comment]
waiting on #67

**Comment by yiwen-h** [comment]
#69  - have done basic sklearn pipeline and tf pipeline. Pretty decent results with SVM which could be further finetuned. TF neural networks probably not worth it.
